mirascope/base/extractors.py

```diff
@@ -481,6 +474,7 @@ class TempCall(call_type):  # type: ignore
             base_url = self.base_url
             api_key = self.api_key
             call_params = self.call_params
+            configuration = self.configuration
```

Comment:
hmm not sure off the top of my head, for now I added a TODO

Reply:
Oh, I guess technically we don't need to do that because the `call_type` will already have the correct provider.

You can remove the TODO and resolve this.